CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ if( BUILD_FAT_LIBMLIRMIOPEN )
   set(BUILD_SHARED_LIBS OFF CACHE BOOL "")
   set(LLVM_BUILD_LLVM_DYLIB OFF CACHE BOOL "")
   set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE STRING "")
+  set(LLVM_ENABLE_ZLIB "OFF" CACHE STRING "")
```

Comment:
Please move it up to around line 16. This shouldn't be used either in the library or driver/runner.

Also, please add the below back:
https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blob/b0d60d9e943c3173e6665f35c6b84ebcc1b1c0bc/CMakeLists.txt#L10